ceph_deploy/install.py

```diff
@@ -37,7 +51,16 @@ def install(args):
 
     for hostname in args.host:
         LOG.debug('Detecting platform for host %s ...', hostname)
-        distro = hosts.get(hostname, username=args.username)
+        distro = hosts.get(
+            hostname,
+            username=args.username,
+            # XXX this should get removed once ceph packages are split for
+            # upstream. If default_release is True, it means that the user is
+            # trying to install on a RHEL machine and should expect to get RHEL
+            # packages. Otherwise, it will need to specify either a specific
+            # version, or repo, or a development branch.
+            use_rhceph=args.default_release,
```

Comment:
correct, I will append an explanation of that case, this might be confusing later on because we are just hijacking the flag to help aid the logic